content/c/concepts/switch/switch.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: 'Switch' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Evaluates an expression or variable to execute a code of block if the expression's result matches a case value.' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Control Flow'
+  - 'If'
+  - 'Else'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+In C, the **`switch case`** statement is one that allows us to execute a block of code, like you would if were to use the `if..else-if..else` statement.  A main difference here, though, is that this statement is much easier to read and write.
```

Comment:
```suggestion

In C, the **`switch case`** statement is one that allows us to execute a block of code, similar to the `if..else-if..else` statement.  A main difference here, though, is that this statement is much easier to read and write.
```

Reply:
Done!  Thank you for that extra revision.